packages/lambdas/src/fitbit.ts

```diff
@@ -0,0 +1,78 @@
+import { getSecret } from "@aws-lambda-powertools/parameters/secrets";
+import * as Sentry from "@sentry/serverless";
+import axios from "axios";
+import { createHmac } from "crypto";
+import { Request } from "express";
+import status from "http-status";
+import { capture } from "./shared/capture";
+import { getEnvOrFail } from "./shared/env";
+
+// Keep this as early on the file as possible
+capture.init();
+
+const apiServerURL = getEnvOrFail("API_URL");
+
+const api = axios.create();
+
+const buildResponse = (status: number, body?: unknown) => ({
+  statusCode: status,
+  body,
+});
+
+const defaultResponse = () => buildResponse(status.NO_CONTENT);
+
+//eslint-disable-next-line @typescript-eslint/no-explicit-any
+// type Request = { body?: any; headers: Record<string, string | undefined> };
+
+export const handler = Sentry.AWSLambda.wrapHandler(async (req: Request) => {
+  console.log(`Verifying at least one UserAuthToken on body...`);
+
+  if (!req.body) {
+    console.log("Resquest has no body - will not be forwarded to the API");
+    return defaultResponse();
+  }
+
+  const fitbitClientSecretName = getEnvOrFail("FITBIT_CLIENT_SECRET");
```

Comment:
let's do this at the top of the lambda so it fails to spin up if this isn't configured properly